napalm/junos/utils/junos_views.yml

```diff
@@ -104,7 +104,7 @@ junos_lldp_table:
 junos_lldp_view:
   fields:
     hostname: lldp-remote-system-name
-    port: lldp-remote-port-description | lldp-remote-port-id
```

Comment:
`lldp-remote-port-description` needs to stay as that's what older Junos versions provide.